Extension/src/Debugger/configurationProvider.ts

```diff
@@ -93,8 +92,8 @@ class CppConfigurationProvider implements vscode.DebugConfigurationProvider {
 	 * Returns a list of initial debug configurations based on contextual information, e.g. package.json or folder.
 	 */
     async provideDebugConfigurations(folder: vscode.WorkspaceFolder | undefined, token?: vscode.CancellationToken): Promise<vscode.DebugConfiguration[]> {
-        let buildTasks: vscode.Task[]; // = await getBuildTasks(); // TODO
-        if (!buildTasks || buildTasks.length === 0) {
+        let buildTasks: vscode.Task[] = await getBuildTasks(); // TODO
```

Comment:
You should try/catch this.  If the promise is rejected, you should get an exception here, not an empty array.

Reply:
The promise is never rejected, right?